com.unity.mobile.notifications/Tests/Runtime/Android/AndroidNotificationSimpleTests.cs

```diff
@@ -3,11 +3,43 @@
 using UnityEngine;
 using UnityEngine.TestTools;
 using Unity.Notifications.Android;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
 
 class AndroidNotificationSimpleTests
+    : IPrebuildSetup, IPostBuildCleanup
 {
     const string kChannelId = "SerializeDeserializeNotificationChannel";
 
+#if UNITY_EDITOR
+    PluginImporter GetTestUtils()
+    {
+        var assets = AssetDatabase.FindAssets("UnityNotificationTestUtils");
+        if (assets.Length != 1)
+            throw new Exception("Something is wrong");
+        return (PluginImporter)AssetImporter.GetAtPath(AssetDatabase.GUIDToAssetPath(assets[0]));
+    }
+#endif
+
+    public void Setup()
+    {
+#if UNITY_EDITOR
+        var testUtils = GetTestUtils();
+        testUtils.SetCompatibleWithPlatform(BuildTarget.Android, true);
```

Comment:
can you explain why this is needed, meaning why UnityNotificationTestUtils is not commited to be always compatible with Android?

Reply:
Because then it is always copied into built player, while we only need it for running tests.